### PR TITLE
Add piedmont piano

### PR DESCRIPTION
**Pull request by nato**

### CODE REVIEW

File: upcoming.md [hunk 1]
```diff
@@ -20,3 +20,6 @@ November 2, 2016 at 7:30pm, Berkeley
 
 November 3, 2016 at 7pm, Oakland
 : [Minsky Duo at Octopus Literary Salon](https://oaklandoctopus.org/minsky-duo-daniel-tortoledo-brooklyn-7pm)
+
+February 26, 2017, Oakland
```
Comment (monicachew) [inline upcoming.md]: We are now at 4pm

File: upcoming.md [hunk 2]
```diff
@@ -20,3 +20,6 @@ November 2, 2016 at 7:30pm, Berkeley
...
+: [Minsky Duo at Piedmont Piano](//piedmontpiano.com/Webpages/11concertpage.html)
```
Comment (monicachew) [inline upcoming.md]: http?

**Comment by monicachew** [comment]
Since you're touching this file, mind updating the orinda groupmuse? It's now October 28.

https://www.groupmuse.com/events/4053-minsky-duo-goes-to-russia

**Comment by monicachew** [comment]
Btw, I made this a table on my site and I think it looks a lot cleaner. I can do it in a later PR though. http://www.monicachew.com/upcoming/

File: upcoming.md (hunk 2 above)
Comment (nato) [inline upcoming.md]: Ya, I leave protocols out:

http://stackoverflow.com/questions/4831741/can-i-change-all-my-http-links-to-just
https://tools.ietf.org/html/rfc3986#section-4.2

File: upcoming.md (hunk 1 above)
Comment (nato) [inline upcoming.md]: Just wanted to double-check, 4pm for the Piedmont show?

Comment (monicachew) [inline upcoming.md]: Yep, 4pm for Piedmont.

On Thu, Aug 25, 2016 at 5:57 PM, Nathanael Bartley <notifications@github.com

**Comment by nato** [comment]
Maybe double-check the time (4pm) -- I was a bit confused.

**Comment by monicachew** [comment]
Thanks!

**Pull request merged**